CoEpi/ble/Peripheral.swift

```diff
@@ -104,24 +113,25 @@ extension Peripheral: CBPeripheralManagerDelegate {
         request.value = CENData
         peripheral.respond(to: request, withResult: .success)
         
+        addNewContactEvent(with: currentCENKey.cenKey!)
+        
         os_log("Peripheral manager did respond to read request with result: %d", log: blePeripheralLog, CBATTError.success.rawValue)
     }
     
-    /* TODO: Later
+
     private func addNewContactEvent(with identifier: String) {
         print("PERIPHERAL: addNewContactEvent called")
-        delegate?.onPeripheralContact(CEN(
-            identifier: identifier,
-            timestamp: Date(),
-            // TODO preference, from React Native
-            isPotentiallyInfectious: true
-        ))
+        delegate?.onPeripheralContact(CEN(CEN: identifier))
```

Comment:
Remove?

Reply:
Idea is to keep both: rx relays and a delegate, for the time being (as suggested by @i-schuetz ).

Reply:
I see now the cause of the confusion. This is not what I meant! I mean that if we have to work with delegates (because the covidwatch library most likely will not support RX), we can bridge those to RX by wrapping them in an adapter.

For now the plan of action is to just stick to one of the models. We agreed on RX. So delegate has to be removed.

Reply:
I see now the cause of the confusion. This is not what I meant! I mean that if we have to work with delegates (because the covidwatch library most likely will not support RX), we can bridge those to RX by wrapping them in an adapter.

For now the plan of action is to just stick to one of the models. We agreed on RX. So delegate has to be removed.

Reply:
Ok. Will make the change in #40